cmd/kube-controller-manager/app/options/options.go

```diff
@@ -77,6 +78,7 @@ func NewCMServer() *CMServer {
 			HorizontalPodAutoscalerSyncPeriod:               metav1.Duration{Duration: 30 * time.Second},
 			HorizontalPodAutoscalerUpscaleForbiddenWindow:   metav1.Duration{Duration: 3 * time.Minute},
 			HorizontalPodAutoscalerDownscaleForbiddenWindow: metav1.Duration{Duration: 5 * time.Minute},
+			HorizontalPodAutoscalerTolerance:                podautoscaler.DefaultTolerance,
```

Comment:
whoops, I missed this the last time around.  Normally, this location is the place for "defaults".  We actually probably just want the default value here (i.e. just have the value here, and don't reference `podautoscaler`, and the rename `DefaultTolerance` to something like `defaultTestingTolerance`...

Reply:
Ah gotcha, I wasn't sure about that. Thanks for catching.